shift_detector/checks/word_prediction_check.py

```diff
@@ -46,23 +46,33 @@ def run(self, store) -> Report:
             logger.info('Automatically selected columns [{}] to be tested by WordPredictionCheck'.format(self.columns))
 
         result = {}
+        failure_columns = {}
         for col in self.columns:
-            result[col] = store[WordPredictionPrecalculation(col,
-                                                             ft_window_size=self.ft_window_size,
-                                                             ft_size=self.ft_size,
-                                                             ft_workers=self.ft_workers,
-                                                             ft_seed=self.ft_seed,
-                                                             lstm_window=self.lstm_window,
-                                                             verbose=0)]
+            try:
+                result[col] = store[WordPredictionPrecalculation(col,
+                                                                 ft_window_size=self.ft_window_size,
+                                                                 ft_size=self.ft_size,
+                                                                 ft_workers=self.ft_workers,
+                                                                 seed=self.seed,
+                                                                 lstm_window=self.lstm_window,
+                                                                 verbose=0)]
+            except ValueError as e:
+                failure_columns[col] = e
+                logger.warning('Skipping textual column {} due to the following error: {}'.format(col, e))
 
         examined_columns = self.columns
-        shifted_columns, explanation = self.detect_shifts(examined_columns, result)
+        columns_no_failure = list(set(examined_columns) - failure_columns.keys())
+        shifted_columns, explanation = self.detect_shifts(columns_no_failure, result)
+
+        for col, err in failure_columns.items():
+            explanation[col] = err
 
         return Report("WordPredictionCheck", examined_columns, shifted_columns, explanation)
 
     def detect_shifts(self, examined_columns: List[str], result: dict):
         shifted_columns = []
         explanation = {}
+        print('detect_shifts')
```

Comment:
this could be something for our new custom print methods (e.g. lprint)

Reply:
I don't think, that this statement has any value to the user

Reply:
just forgot to remove it